docs/add-an-experiment.rst

```diff
@@ -30,9 +30,21 @@ and package/application repositories, are used to generate a set of concrete Ram
 
 Some or all of the functions in the Experiment base class can be overridden to define custom behavior, such as adding experiment variants. 
 
+compute_package_section
+~~~~~~~~~~~~~~~~~~~~~~~
+In ``compute_package_section`` add the benchmark's package spec. Required packages for the benchmark should be defined in the ``package.py``. 
+
+::
+  
+  def compute_package_section(self):
+      self.add_package_spec(self.name, [f"BENCHMARK@APP_VERSION [ADDITIONAL_SPECS]"])
```

Comment:
You are missing the line on application version here. 

The explanation needs to explain what additional specs means - maybe an example with +mpi could make this more clear.

Reply:
Done